dataplug-instagram/app/com/hubofallthings/dataplugInstagram/apiInterfaces/InstagramFeedInterface.scala

```diff
@@ -146,6 +153,25 @@ class InstagramFeedInterface @Inject() (
     }
   }
 
+  private def transformData(rawData: JsValue): JsResult[JsObject] = {
+    val prependFieldsId = configuration.get[String]("service.customFieldId")
+    val transformation = (__ \ "data").json.update(
+      __.read[JsArray].map { feedData =>
+        {
+          val updatedFeedData = feedData.value.map { item =>
+            val unixTimeStamp = DateTime.parse((item \ "timestamp").asOpt[String].getOrElse("")).getMillis / 1000
+            item.as[JsObject] ++ JsObject(Map(
+              s"${prependFieldsId}_api_version" -> JsString("v2"),
+              "created_time" -> JsString(unixTimeStamp.toString)))
```

Comment:
This has to be the same as v1. Or I copy paste the instagram mapper on HAT and we have 2